CFC_WebApp/clients/socialmedia/socialmedia.py

```diff
@@ -0,0 +1,55 @@
+"""
+Client module to be used as test gateway for social media authorization
+
+Instructions:
+
+    - Get key values from Sid and `cp settings.json.sample settings.json`
+
+"""
+import datetime
+import json
+from os.path import dirname
+from dao.socialmediauser import FacebookUser
+from get_database import get_client_db
+
+FB_BASE_DIR = dirname(dirname(__file__))
+
+
+def getFacebookUser(cookies):
+    return FacebookUser.get_user_from_cookies(cookies)
+
+
+def facebookLogin():
```

Comment:
True… was going to defer messing w/choice until I got Cordova set up.




> On Jun 2, 2015, at 11:28 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/clients/socialmedia/socialmedia.py <https://github.com/e-mission/e-mission-server/pull/133#discussion_r31596730>:
> 
> > +
> > +"""
> > +import datetime
> > +import json
> > +from os.path import dirname
> > +from dao.socialmediauser import FacebookUser
> > +from get_database import get_client_db
> > +
> > +FB_BASE_DIR = dirname(dirname(__file__))
> > +
> > +
> > +def getFacebookUser(cookies):
> > +    return FacebookUser.get_user_from_cookies(cookies)
> > +
> > +
> > +def facebookLogin():
> Hm, I see what you are trying to do here. If you want to display this screen as one of the tabs in the result screen, you need to modify the choice screen to add it. Otherwise, this is code that is never exercised, specially since we are going to remove the API endpoint.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e-mission/e-mission-server/pull/133/files#r31596730>.
>